src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -148,7 +148,7 @@ class AdoConnector {
             };
 
             foreach (var workItemReference in (await _client.QueryByWiqlAsync(wiql)).WorkItems) {
-                var item = await _client.GetWorkItemAsync(_project, workItemReference.Id, expand: WorkItemExpand.Fields);
+                var item = await _client.GetWorkItemAsync(_project, workItemReference.Id, expand: WorkItemExpand.All);
```

Comment:
We need to get work item relations now but for some reason `WorkItemExpand.Fields | WorkItemExpand.Relations` didn't work. Possibly relevant link? https://github.com/microsoft/azure-devops-dotnet-samples/issues/271